### PR TITLE
Eccentricity Documentation

### PR DESCRIPTION
**Pull request by egolol**
#173 Eccentricity Documentation

### CODE REVIEW

File: braph/graph/measures/Eccentricity.m [hunk 1]
```diff
@@ -1,12 +1,49 @@
 classdef Eccentricity < Measure
+    % Eccentricity < Measure: Eccentricity measure
+    % Eccentricity provides the eccentricity of a node for binary undirected
+    % (BU) and weighted undirected (WU) graphs. It is calculated as the 
+    % maximal shortest path length between a node and any other node.
+    % 
+    % Eccentricity methods:
+    %   Eccentricity                      - constructor with Measure properties.
```
Comment (maverickyuwei) [inline braph/graph/measures/Eccentricity.m]: perhaps reduce tab or space to indent the code?

File: braph/graph/measures/Eccentricity.m [hunk 2]
```diff
@@ -1,12 +1,49 @@
...
+    %   getCompatibleGraphNumber    - returns the number of compatible graphs.
+    %
+    % See also Measure, Graph, Strength, Distance, Degree.
     methods
         function m = Eccentricity(g, varargin)
+            % ECCENTRICITY(G) creates eccentricity with default measure properties.
+            % G is a graph (e.g, an instance of GraphBD, GraphBU,
+            % GraphWD, Graph WU). 
+            %   
+            % ECCENTRICITY(G, 'Settings', SETTINGS) ceates eccentricity measure and
```
Comment (annacanal) [inline braph/graph/measures/Eccentricity.m]: creates

Comment (annacanal) [reply braph/graph/measures/Eccentricity.m]: I just checked, it is also a problem in Degree, which typo already. If you can change this in Degree constructor as well in this same branch.

File: braph/graph/measures/Eccentricity.m [hunk 3]
```diff
@@ -1,12 +1,49 @@
...
+            
             settings = clean_varargin({'EccentricityRule'}, varargin{:}); 
             m = m@Measure(g, settings{:});
         end
     end
     methods (Access = protected)
         function ecc = calculate(m)
+            % CALCULATE calculates the eccentricity value of a node
+            %
+            % eccentricity = CALCULATE(M) returns the value of the eccentricity of a
```
Comment (annacanal) [inline braph/graph/measures/Eccentricity.m]: Shouldn't we call the variable eccentricity instead of ecc ?

**Comment by egolol** [comment]
thank you guys for reviewing. :)

File: braph/graph/measures/Eccentricity.m [hunk 4]
```diff
@@ -1,12 +1,49 @@
...
+    %
+    % See also Measure, Graph, Strength, Distance, Degree.
     methods
         function m = Eccentricity(g, varargin)
+            % ECCENTRICITY(G) creates eccentricity with default measure properties.
+            % G is a graph (e.g, an instance of GraphBD, GraphBU,
+            % GraphWD, Graph WU). 
+            %   
+            % ECCENTRICITY(G, 'Settings', SETTINGS) creates eccentricity measure and
+            % initializes the property settings with SETTINGS.
```
Comment (giovannivolpe) [inline braph/graph/measures/Eccentricity.m]: Add what are the possible settings

File: braph/graph/measures/InEccentricity.m [hunk 5]
```diff
@@ -1,12 +1,48 @@
...
+    %
+    % See also Measure, Graph, Strength, Distance, Efficency.
     methods
         function m = InEccentricity(g, varargin)
+            % INECCENTRICITY(G) creates in-eccentricity with default measure properties.
+            % G is a graph (e.g, an instance of GraphBD, GraphBU,
+            % GraphWD, Graph WU). 
+            %   
+            % INECCENTRICITY(G, 'Settings', SETTINGS) creates in-eccentricity measure and
+            % initializes the property settings with SETTINGS.
```
Comment (giovannivolpe) [inline braph/graph/measures/InEccentricity.m]: Add settings explicitly

File: braph/graph/measures/OutEccentricity.m [hunk 6]
```diff
@@ -1,12 +1,47 @@
...
+    %
+    % See also Measure, Graph, Strength, Distance, Efficency.
     methods
         function m = OutEccentricity(g, varargin)
+            % OUTECCENTRICITY(G) creates out-eccentricity with default measure properties.
+            % G is a graph (e.g, an instance of GraphBD, GraphBU,
+            % GraphWD, Graph WU). 
+            %   
+            % OUTECCENTRICITY(G, 'Settings', SETTINGS) creates out-eccentricity measure and
+            % initializes the property settings with SETTINGS.
```
Comment (giovannivolpe) [inline braph/graph/measures/OutEccentricity.m]: Add settings explicitly

File: braph/graph/measures/InEccentricity.m [hunk 7]
```diff
@@ -1,12 +1,51 @@
...
         function m = InEccentricity(g, varargin)
+            % INECCENTRICITY(G) creates in-eccentricity with default measure properties.
+            % G is a graph (e.g, an instance of GraphBD, GraphBU,
+            % GraphWD, Graph WU). 
+            %   
+            % INECCENTRICITY(G, 'Settings', SETTINGS) creates in-eccentricity measure and
+            % initializes the property settings with SETTINGS. INECCENTRICITY
+            % available SETTINGS are: 
+            % subgraphs     -   Calculates INECCENTRICITY of all subgraphs.
+            % default       -   Calculates INECCENTRICITY of global graph.            %
```
Comment (giovannivolpe) [inline braph/graph/measures/InEccentricity.m]: % 'InEccentricityRule' = 'subgraphs' (default) - calculates INECCENTRICITY within each connected subgraph.
            %                                      'default' - calculates INECCENTRICITY of global graph.

**Pull request merged**